docs/src/architecture/state.md

```diff
@@ -13,26 +12,26 @@ The Miden Node(s) maintain three databases to describe the state:
 
 ## State components
 
-These databases are represented by authenticated data structures, such that we can easily prove that items were added to or removed from a database, and a commitment to the database would be very small.
+These databases are represented by authenticated data structures, which allows us to quickly show that objects were added to or withdrawn from a database, and a commitment to the database is minimal.
 
-### Account database
-Current account states are recorded in a Sparse Merkle tree which maps account IDs to account hashes, where the account hash is computed as 
+### Account DB
+Current account states are recorded in a Sparse Merkle tree which maps account IDs to account hashes, where the account hash is computed as:
 
 `hash([account ID, 0, 0, nonce], [vault root], [storage root], [code root])`.
 
 <p align="center">
   <img src="../diagrams/architecture/state/Account_DB.png">
 </p>
 
-As described in [accounts](https://0xpolygonmiden.github.io/miden-base/architecture/accounts.html), there are two types of accounts:
+As described in [this document](https://0xpolygonmiden.github.io/miden-base/architecture/accounts.html), there are two types of accounts:
 * **Public accounts** where all account data is stored on-chain.
 * **Private accounts** where only the hashes of accounts are stored on-chain.
 
 > Losing a state of a private account would mean loss of funds (as the user won't be able to execute transactions) in a similar manner as a loss of a private key would. This problem can be easily mitigated by storing encrypted account state in a cloud or backing it up somewhere else. Unlike storing private keys in the cloud, this does not compromise privacy or security of an account.
 
-Note: Having many (or even most) of the accounts be private is very beneficial for the network as a private account contributes only 40 bytes to the global state (8 bytes account ID + 32 bytes account hash). Or, said another way, 1 billion private accounts takes up only $40$ GB of state.
+Note: Having many (or even most) of the accounts be private is very beneficial for the network as a private account contributes only 40 bytes to the global state (8 bytes account ID + 32 bytes account hash). Simply put, 1 billion private accounts takes up only $40$ GB of state.
 
-### Note database 
+### Notes DB
```

Comment:
I think we should always use singular here as well, `Note DB` instead of `Notes DB`. That way it is more consistent with `Account DB`.

Reply:
Gotcha!